maat-court-data-api/src/test/java/gov/uk/courtdata/integration/applicant/ApplicantServiceIntegrationTest.java

```diff
@@ -68,6 +70,45 @@ void givenInValidRepId_WhenGetRepOrderApplicantLinksIsInvoked_thenExceptionIsRai
                 .hasMessageContaining("Rep Order Applicant Links not found for repId");
     }
 
+    @Test
+    void givenAValidInput_whenUpdateRepOrderApplicantLinksIsInvoked_thenUpdateIsSuccess() {
```

Comment:
Now that we're checking the json in the controller, do we really need this test class? I think I'd still advocate for combining the two. This first test cases is at least pretty much identical to the one in the controller.

Reply:
I feel this is required for the segregation of responsibilities. Combining 2 mixes controller and service layer.